recipes/pytorch-msssim/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pytorch-msssim" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorch_msssim-{{ version }}.tar.gz
+  sha256: 53273f03ce5c435f706f49feb5110a8afe90b2519484b0fe0df9639670106d0b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [win]
```

Comment:
We prefer noarach package even if it fails on Windows due to the lack of pytorch. Windows wouldn't be installable anyway and, when pytorch is available there, this package would be ready without the need for any change.

Reply:
Reverted back to version with noarch specification, thank you for letting me know